Jenkinsfile

```diff
@@ -1,16 +1,8 @@
-def buildNumber = BUILD_NUMBER as int; if (buildNumber > 1) milestone(buildNumber - 1); milestone(buildNumber) // JENKINS-43353 / JENKINS-58625
```

Comment:
why are you removing this?

Reply:
Because I have no idea what is it for, and it's documented nowhere?

Reply:
it's not, it's a @jglick jenkins hack to do with parallel builds and aborting old ones when new ones come in or something along those lines

Reply:
Oh I see: it explains why my builds where always killed 😅 
Then removing this line is a feature then: it allows to not waste builds when doing extensive tests (given the build time here: I would prefer not killing previous builds).

WDYT?

Reply:
(I would also add that by switching to Kubernetes pods instead of VMs, we are not wasting a lot of resource with multiple builds now, WDYT?)

Reply:
Up to you. When builds are long and resources are limited, aborting outdated builds is generally best.

Reply:
Good catch @timja @jglick ❤️ 

With the explanation from both of you, I confirm that the PR is kept with this deletion.

Reply:
It was introduced in https://github.com/jenkins-infra/packer-images/pull/39

I think because of lots of packer builds holding a lot of executors in ci.jenkins.io...

Reply:
@timja yup, thanks! I also ended on the same PR (39) from my research to understand a bit better. But as the current PR also introduces kubernetes agent, the executor limitation is not there anymore